py/tests/pathfinder_worker/test_call.py

```diff
@@ -132,10 +134,7 @@ def test_command_parsing_call():
     assert command.has_pending_data()
 
 
-@pytest.mark.skip(
-    reason="this is not a test but utility function working around pytest"
-)
-def test_relative_path(path):
+def check_relative_path(path):
```

Comment:
To me utility method (that is a part of a test set) that starts with 'test_' and thus requires a custom annotation not to be treated as a test (and being show as skipped on every run of the test set) does not make much sense.

To me also "check that file [at a given absolute path] exists" makes more sense than "test" the file. What kind of test is that where no assertions are made?

But I'm fine either way - I will put the old name back, as well as 'skip' annotation.

Reply:
Yes, I know that because the `test_` prefix it would be handled as a test case.

However, the point of that function is not really _checking_ if the file exists, but instead _returning_ the absolute path of the file relative to the test file.

Don't get me wrong, I'm not saying that the original was a perfect name -- I'm sure we could come up with something better. I'm just saying that the new name is really misleading to me because it's not really a check.

Reply:
Sorry for bitching about the "bike rack", just "purist mode" sometimes takes over :) I don't really care how it is called while it is doing it's job. Naming things is always tricky.

Reply:
Instead of having a `testdir()` function we can have `TEST_PATH` as a value. Something like this:

```
TEST_PATH = pathlib.Path(__file__).parent

[...]

    path = TEST_PATH.joinpath(
        "../../../crates/gateway-test-fixtures/fixtures/contracts/contract_definition.json.zst"
    )
```

This way we don't even need a function.

Reply:
Or we can also call it `TEST_DIR` of course, since it's the path to the directory of the test.

Reply:
Many options :D I haven't actually checked the usage so maybe this doesn't work well. But just pick one and lets get this PR done :)

```
# Whicher of these two works:
PROJECT_ROOT = pathlib.Path(__file__).parent.parent.parent.parent
PROJECT_ROOT = pathlib.Path(__file__).parent.join("../../..")

[...]

    path = PROJECT_ROOT.joinpath(
        "crates/gateway-test-fixtures/fixtures/contracts/contract_definition.json.zst"
    )
```